bin/sl-run.js

```diff
@@ -17,6 +17,14 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var tracer = require('concurix');
```

Comment:
It was made parallel to 'var agent.'  Shall we make just "var trace" conditional on config.profile or make both agent declaration and tracer declaration conditional on config.profile?